azavea/echo-locator
# Do not show centroid markers

## Overview

 - Moves hover and click interactivity to neighborhood bounds layer.
 - Styles neighborhood bounds differently for routable and non-routable neighborhoods.
 - Removes centroid marker layer.


### Demo

On hover, boundary is green and town label displays. Routable neighborhood bounds are blue, non-routable neighborhood bounds are grey (same as the centroids were):
![echo_hover_green_boundary](https://user-images.githubusercontent.com/960264/56531786-9a16f500-6522-11e9-926a-23681cc7f100.png)


## Testing Instructions

 * Clicking a routable neighborhood boundary on the map should go to details
 * Neighborhood bounds map hover should highlight bounds and show label
 * Going to details from sidebar map marker click or sidebar card click should still work
 * Returning from list should unset the active neighborhood (hide the detail route on map)

Closes #125.
